wrapping/test/convert_test.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
@dzenanz The license header is missing in this file.

Reply:
Done via db9eb2e4b6025337d72009b643d1700d51c36fe5.